src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.css

```diff
@@ -24,6 +24,24 @@ dashboard-page .dashboard-panel .tab-header .action-item .action-label.close {
 	padding: 5px;
 }
 
+.home-tab-icon {
```

Comment:
>.home-tab-icon [](start = 0, length = 14)

These should really be scoped under dashboard-page at least (try to scope them down as much as makes sense)